### DISCUSSION TITLE
不义之财更新后将关卡名设置成新的关卡会卡住

### DISCUSSION THREAD

**Discussion by yiranwang0504** [MaaAssistantArknights/MaaAssistantArknights#6258] [Q&A]
![fade8df6f969dd17fbec7407cca6448a](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/91300103/fce22495-eb51-4198-be7d-c4b7ea6500c8)
如图，macbook版MAA将关卡名设置成cv关卡会卡在这里，设置成当前/上一次能解决

**Comment by ABA2396** [comment]
mac是这么访问api的？ @hguandl 瞅瞅）

**Reply by hguandl** [reply]
macOS 版目前没有实现 resource 热更新，CV 的关卡导航得等发新版本才会有。该新建这个功能的文件夹了（
